code-challenges.js

```diff
@@ -15,35 +15,128 @@ const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
 
+
+
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+// Pseudo:
+// - create a function called boilingPoint
+// - input - temp
+// - use conditional statement to make an evaluation on the temperature
+// - if temp is at or below 42, output temp is below boiling point
+// - if temp is at or above 212, output temp is at boiling point
+// - if temp is at or above 350, output temp is above boiling point
+
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
 
+const boilingPoint = (temp) => {
+  if(temp <= temp1){
+  return `${temp} is below boiling point`
+} else if (temp >= temp2){
+  return `${temp} is above boiling point`
+} else if (temp >= temp3){
+  return `${temp} is at boiling point`
+} else if(temp > temp1){
+  return `${temp} is close to boiling point`
+}else {
+  return ""
+  }
+}
+console.log(boilingPoint(41))
+console.log(boilingPoint(42))
+console.log(boilingPoint(100))
+console.log(boilingPoint(212))
+console.log(boilingPoint(250))
+console.log(boilingPoint(350))
+console.log(boilingPoint(351))
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+// // Pseudo:
+// - create a variable allNumbers
+// - use the concat method to join myNumbers1 and myNumbers2 to equal allNumbers variable
+// - console.log the allNumbers variable with .length to output the length of allNumbers
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+const allNumbers = myNumbers1.concat(myNumbers2)
+console.log(allNumbers.length)
+
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
+// Pseudo:
+// - create variable currentCohortReverse
+// - make currentCohortReverse to equal currentCohort
+// - use the split method to turn the string into an array of substrings
+// - use the reverse method to reverse the substrings
+// - use the join method pull the substrings back together as 1 string
+
 const currentCohort = "Bravo 2022"
 
+const currentCohortReverse = currentCohort.split(" ").reverse().join(" ")
+
+console.log(currentCohortReverse)
```

Comment:
Logic looks great, it looks like this would output "2022 Bravo" though.  This is because adding the space inside the .split will break up the array to look like this: `[ 'Bravo', '2022' ]`.  If you write it .split(""), this will break up the array to look like: `[ 'B', 'r', 'a', 'v', 'o', ' ', '2', '0', '2', '2' ]` allowing .reverse() to reverse all characters.